ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerDefinitionExpression/FeatureLayerDefinitionExpression.cpp

```diff
@@ -84,8 +88,40 @@ bool FeatureLayerDefinitionExpression::layerInitialized() const
   return m_initialized;
 }
 
-void FeatureLayerDefinitionExpression::setDefExpression(QString whereClause)
+int FeatureLayerDefinitionExpression::currentFeatureCount_() const
 {
-  // In QML, "req_Type = \'Tree Maintenance or Damage\'"
+  return m_currentFeatureCount;
+}
+
+void FeatureLayerDefinitionExpression::setDefExpression(const QString& whereClause)
+{
+  // In QML, "req_type = \'Tree Maintenance or Damage\'"
   m_featureLayer->setDefinitionExpression(whereClause);
+  updateFeatureCount();
+}
+
+void FeatureLayerDefinitionExpression::setDisplayFilter(const QString& whereClause)
+{
+  auto displayFilter = new DisplayFilter("Damaged Trees", whereClause);
+  QList<DisplayFilter*> available_filters{displayFilter};
+
+  ManualDisplayFilterDefinition* display_filter_defintion = new ManualDisplayFilterDefinition(displayFilter, available_filters);
+  m_featureLayer->setDisplayFilterDefinition(display_filter_defintion);
+
+  updateFeatureCount();
+}
+
+void FeatureLayerDefinitionExpression::updateFeatureCount()
```

Comment:
🔧  I recommend renaming this method. To a GIS newbie, this could be interpreted many ways. Perhaps something like `queryFeatureCountInCurrentExtent`? It's a mouth full, but gets the point across about precisely what you're doing.